### PR TITLE
Fuse box

### PR DESCRIPTION
**Pull request by evilchia**
Using fuse-box.js as the application bundler.  All major functionality available with minor workarounds.  NPM scripts all work.  .d.ts files not manifesting yet because of anticipated mis/non-configuration

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,60 +1,74 @@
...
+	"name": "cydran",
+	"version": "0.0.24",
+	"description": "Unobtrusive Javascript presentation framework",
+	"main": "dist/cydran.js",
+	"types": "dist/cydran.d.ts",
+	"scripts": {
+		"start": "node fuse",
+		"build": "node fuse build && npm run minify",
+		"releaseBuild": "npm run test && npm run lint && node fuse release",
+		"clean": "node fuse clean",
```
Comment (matthewwerny) [inline package.json]: We still need to delete the dist folder as well to match the previous and needed clean behavior

**Comment by matthewwerny** [comment]
We also lost the ability to do cydran.d.ts generation with the dts-bundle-generator invocation.  We'll need to figure out how to invoke that from fuse.

**Comment by matthewwerny** [comment]
We'll need to resurrect the invocation of the DTS bundle generator from the webpack configs,as it generates the cydran.d.ts file after the bundle is updated

File: package.json (hunk 1 above)
Comment (evilchia) [reply package.json]: It does delete the dist folder.  Any of the node fuse <activities> have node fuse clean as a prerequisite activity.  See the fuse.js file.

**Pull request merged**